packages/trip-details/src/index.js

```diff
@@ -0,0 +1,145 @@
+import {
+  calculateFares,
+  calculatePhysicalActivity
+} from "@opentripplanner/core-utils/lib/itinerary";
```

Comment:
A note about using .../lib/... paths on import statements:

In a future PR, I'm going to go thru all the files in otp-ui and switch all the .../lib/... import paths to .../src/...  Cole pointed out that by us using transpiled .../lib/... code, we're preventing webpack from tree-shaking the otp-ui code, resulting in larger footprints than might be necessary.  We might do more, based on Cole's continued research in making the code performant.

We're not there yet tho, so no action necessary just yet. More just a heads-up.
We're at the 'get it running phase' right now.
Next, we'll move to 'get it running correctly'.
Then when (if) we get to the 'get it running fast' phase, expect said changes...

Reply:
BTW, I've tried out the .../src/... paths in a dev bundle on npm (https://www.npmjs.com/package/@ott_test/transit-vehicle-overlay), and I was able to pull in base-map, core-utils and transit-vehicle-overlay, which used the .../src/... paths, and my simple_app (https://github.com/trimet-sandbox/simple_app), as well as dev version of trimet.org were able to pull in the code and do a latter transpile in building the app.  More testing necessary, but...

Reply:
Do you want to copy this comment to an issue so that we can track it outside of this PR, @fpurcell?